app/controllers/pending_purchases_controller.rb

```diff
@@ -3,7 +3,7 @@ class PendingPurchasesController < ApplicationController
   before_action :set_pending_purchase, only: [:edit, :update]
 
   def index
-    @pending_purchases = PendingPurchase.where(user: current_user)
+    @pending_purchases = PendingPurchase.pending(current_user)
```

Comment:
This works as-is, but I would restructure it a little, and do `current_user.pending_purchases.pending`. That will require changing your `pending` method to a scope, and not passing the user in.

The reason for this is that if you later need to say, find all pending PendingPurchases, you can just do `PendingPurchase.pending`.